### PR TITLE
Instructions for usage with Moodle Docker.

### PR DESCRIPTION
**Pull request by paulholden**
Because it took a bit of time to figure out while working on https://tracker.moodle.org/browse/MDL-76291 I thought it best we document how to do this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,17 @@ For example, add the following to your config.php after the `$CFG->wwwroot` line
    define('TEST_MOD_BIGBLUEBUTTONBN_MOCK_SERVER', "http://localhost:8001/hash" . sha1($CFG->wwwroot));
    ```
 
+### Moodle Docker
+To run automated tests in conjunction with the [Moodle Docker](https://github.com/moodlehq/moodle-docker) testing environment you
+should start the mock server as above, then determine the gateway address of the webserver container, e.g.:
+   ```
+   docker inspect -f "{{ .NetworkSettings.Networks.docker_default.Gateway }}" docker-webserver-1
```
Comment (andrewnicols) [inline README.md:26]: Err... what is `docker-webserver-1`?

Typically if you're using this in the moodle-docker container, the simpler option is to add a new yml file which configures this host, and then use the hostname.

Any docker-compose configuration sets up DNS.

**Review by andrewnicols — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (paulholden) [reply README.md:26]: > Err... what is docker-webserver-1?

It's the name of the running container created when you start moodle-docker

**Comment by paulholden** [comment]
Cheers @andrewnicols - admittedly the initial version was just me documenting my own attempt at running the mock server with Moodle - however now that https://github.com/moodlehq/moodle-docker/pull/241 adds in-built support in Moodle Docker, I've simplified the readme additions here to just note that

**Review by paulholden — COMMENTED** [review]
Fixed

**Comment by stronk7** [comment]
LGTM, hope you don't mind @andrewnicols ...

**Review by stronk7 — APPROVED** [review]

**Pull request merged**